site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2449,7 +2498,7 @@ public function getGradedComponentsCountByGradingSections($g_id, $sections, $sec
     public function getBadGradedComponentsCountByGradingSections($g_id, $sections, $section_key, $is_team) {
         //getBadTeamSubmissionsByGradingSection
         //getBadUserSubmissionsByGradingSection
-         $u_or_t = "u";
+        $u_or_t = "u";
```

Comment:
Although this change is a change in a positive direction, it's unrelated to this PR and should be reverted to reduce the risk of conflicts, as well as make the real changes in this PR clear.